sycl/include/CL/sycl/detail/pi.h

```diff
@@ -913,6 +913,9 @@ piextDeviceGetNativeHandle(pi_device device, pi_native_handle *nativeHandle);
 __SYCL_EXPORT pi_result piextDeviceCreateWithNativeHandle(
     pi_native_handle nativeHandle, pi_platform platform, pi_device *device);
 
+/// Returns plug-in specific data
```

Comment:
Please describe the argument, what it returns and how caller interprets that.
Why cannot this opaque pointer be added as a field of _pi_plugin https://github.com/intel/llvm/blob/6a49170027fba61cea649ffdd299ed095b5e3d75/sycl/include/CL/sycl/detail/pi.h#L1608 and just be always filled at plugin's initialization `piPluginInit` API? Why do we need the new API?

Reply:
Agree. I'll remove the new API and include the instantiation of device-specific data in `piPluginInit`.